ui/src/router.ts

```diff
@@ -214,6 +217,23 @@ const routes: Array<RouteConfig> = [
           }
         ],
       },
+      {
+        path: '/hierarchies',
+        name: 'Hierarchies',
+        component: Hierarchies,
+        children: [
+          {
+            path: 'new',
+            name: 'HierarchyCreate',
+            component: HierarchyCreate,
+          },
+          {
+            path: '/hierarchy/:id',
```

Comment:
To follow the other URIs in the app, I would use `path: ':id'` so that the URL reflects the nesting of the UI.